### ISSUE TITLE
German Traffic Sign Classification

### ISSUE THREAD

**Issue by Best-forever-003** [prathimacode-hub/ML-ProjectKart#408]
Define You:

NAME: -AAMIR P
USERNAME: Best-forever-003
PROJECT: German Traffic Sign Classification

- [x] DevIncept Participant 
- [x] Contributor

Can you, please assign me this project Madam? @prathimacode-hub 

**Is your feature request related to a problem? Please describe.**
Yeah related to a traffic system

**Describe the solution you'd like...**

Python and TensorFlow to classify traffic signs.

This dataset has more than 50,000 images of 43 classes. This would make the traffic system more accurate! 

Load The Data.
Dataset Summary & Exploration
turning data into tensors
Getting images and their labels
turning images into tensors
Data Preprocessing.
Normalization
resize
Design a Model Architecture. -CNN Model
Model Training and Evaluation.
Testing the Model Using the Test Set.

**Comment by prathimacode-hub** [comment]
Can you provide me with the dataset you'll be working here? I shall assign this issue on 10th of July, 2021. @Best-forever-003

**Comment by Best-forever-003** [comment]
https://www.kaggle.com/meowmeowmeowmeowmeow/gtsrb-german-traffic-sign @prathimacode-hub

**Comment by prathimacode-hub** [comment]
> https://www.kaggle.com/meowmeowmeowmeowmeow/gtsrb-german-traffic-sign @prathimacode-hub

Good choice of the project. All the best. Make sure you get it assigned then and stay alerted on the concerned day as it's on FCFS basis. @Best-forever-003

**Comment by Best-forever-003** [comment]
Thank you so much Madam @prathimacode-hub

**Comment by prathimacode-hub** [comment]
> Thank you so much Madam @prathimacode-hub

Welcome, I had already assigned DevIncept label, so other program participants doesn't take up this issue. @Best-forever-003

**Comment by prathimacode-hub** [comment]
If you're interested to work on this issue, kindly let us know. @Best-forever-003

**Comment by harshita9621** [comment]
@prathimacode-hub Can I work on this issue under DevIncept?

**Comment by prathimacode-hub** [comment]
Decide of the project you want to work on, this interest is first raised by other person. If he doesn't show up by tomorrow, you can acquire this. There are many other issues up for grabs. About Covid Analysis you had already worked on it once, lets give chance to others. Else, you can come up a new issue. @harshita9621

**Comment by harshita9621** [comment]
@prathimacode-hub Okay

**Comment by Best-forever-003** [comment]
> If you're interested to work on this issue, kindly let us know. @Best-forever-003

Yes Madam, I shall complete this by tomorrow morning @prathimacode-hub

**Comment by Best-forever-003** [comment]
I'm interested in working on this issue @prathimacode-hub Madam

**Comment by prathimacode-hub** [comment]
Sure, issue is assigned. Take your time. Max time for any issue to complete is 5days. Also intimate your progress once in 2days. @Best-forever-003
